src/NServiceBus.Persistence.CosmosDB/NServiceBus.Persistence.CosmosDB.csproj

```
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Particular.Packaging" Version="1.4.0" PrivateAssets="All" />
     <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.1900, 9.0.0)" />
-    <PackageReference Include="Microsoft.Azure.Cosmos" Version="[3.20.1, 4)" />
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.23.0" />
```

```suggestion
    <PackageReference Include="Microsoft.Azure.Cosmos" Version="[3.20.1, 4)" />
```

or we can enforce the new minor which would be OK if we need a specific functionality

```suggestion
    <PackageReference Include="Microsoft.Azure.Cosmos" Version="[3.23.0, 4)" />
```

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/changelog.md

3.23 adds support for the patch operation, which we might need according to the discussion we had last week.